### ISSUE TITLE
How can you log with tensorboard using the setfit trainer ?

### ISSUE THREAD

**Issue by tomad02** [huggingface/setfit#388]
How can you log with tensorboard using the setfit trainer ?

**Comment by tomaarsen** [comment]
Hello!

I'm afraid this isn't possible currently due to limitations in [SentenceTransformer](https://github.com/UKPLab/sentence-transformers/blob/master/sentence_transformers/SentenceTransformer.py#L575). I'll try to get around this.

- Tom Aarsen

**Comment by grofte** [comment]
I think people are mostly interested in monitoring training of the head model. Or what @tomad02 ? Typically you would train the head multiple epochs and there's a pretty big risk of overfitting when you have 768 features to train on. So you would want to see the loss and metric for the training and eval dataset for those epochs. Of course you could also inspect the weight distribution and so on but I don't know how much information that would give people.

**Comment by tomaarsen** [comment]
#439 will introduce support for Tensorboard (+ Weights and Biases and others) when finetuning the body. It should be released this week, stay tuned!
